src/Account/Session/ShopSession.php

```diff
@@ -35,25 +37,41 @@
 class ShopSession extends Session implements SessionInterface
 {
     /**
-     * @var ShopProvider
+     * @var CommandBus
      */
-    protected $oauth2ClientProvider;
+    protected $commandBus;
 
     /**
      * @var ConfigurationRepository
      */
     protected $configurationRepository;
 
+    /**
+     * @var ShopProvider
+     */
+    protected $oauth2ClientProvider;
+
+    /**
+     * @var LinkShop
+     */
+    protected $linkShop;
```

Comment:
apparently not used anymore, but the `getShopUuid` method is probably useless if we use this member instead.